ExternalFilePrinting.md

```diff
@@ -18,9 +18,11 @@ Here's the flow for extending Ringtail's production printing process:
 
 1. The extension returns:
    - HTTP status code of 200 (OK) if the file is ready and returned in the body of the response.
-   - HTTP status code of 204 (No Content) if the file is not yet ready, indicating that Ringtail should retry the request at a later time.
-      - Use this response code to indicate that the request is being successfully processed but the file is not yet ready.
-      - Ringtail will retry file requests for approximately an hour. After that, Ringtail will mark the document as `Error Printing - External`.
+   - HTTP status code of 202 (Accepted) with a Location header set if the file is not ready yet, indicating that Ringtail should make a request to the specified location at a later time.  see [RFC 7240](https://tools.ietf.org/html/rfc7240#section-4.1)
```

Comment:
I would suggest mentioning this up top, perhaps right after the intro paragraph:

> To retrieve externally printed files, Ringtail supports the asynchronous HTTP request pattern per [RFC 7240](https://tools.ietf.org/html/rfc7240#section-4.1).

Then I'd change this line to:

> HTTP status code of 202 (Accepted) with a `Location` header set if the file is not ready yet, indicating that Ringtail should make a subsequent request to the specified location.